src/client/src/App/Trades/__tests__/FxTrades.filter.test.tsx

```diff
@@ -62,7 +62,7 @@ describe("for notional column", () => {
     expect(container.querySelector(notionalFilterMenu)).toBe(null)
   })
 
-  it("filter icon and menu should work correct", async () => {
+  it.skip("filter icon and menu should work correct", async () => {
```

Comment:
@algreasley - the column filter functionality on the app (including current React 17 / prod deployment) is very broken on date and number columns.  We should review and fix that before re-enabling this test.

Reply:
it's not green on my branch.

Reply:
well, yes 😄 
but I was keeping to the principle of "make the fewest number of changes" when changing something fundamental like this .. I wasn't aware that the grid was broken now .. gift that keeps giving, that one ..

Reply:
grid filter for Trade Date (first thing I tried) appears to be working for me (on Dev and running up your branch) .. what are the issues?